### ISSUE TITLE
Custom Styles are not applied correctly in all cases

### ISSUE THREAD

**Issue by julie777** [stevencohn/OneMore#913]
# Problem to Solve

Custom styles are not applying correctly. 

# To Reproduce

I noticed today that when I applied "Heading 1" it looked small. When I checked my current style settings, in fact heading 1 was set to Open Sans 18, but my heading was Calibri 16.

After going through the entire page clicking in each heading and using `<ctrl><alt><shift>-1` to update the style each update changed the size to 18. However the font is still set to Calibri.

Next I tried "apply these styles to page" and all styles were set to match mynormal.

So why did full page work and not just the heading?

I just manually set the font for a heading to Calibri and then used the shortcut for Heading 1 and it worked correctly.

# Expected Behavior

The current paragraph should be styled to Custom Heading 1 settings after I apply the style.

# Screenshots

![image](https://user-images.githubusercontent.com/5090005/219140502-4eaae9c7-5098-4c38-baf7-73b30e2a20e2.png)

![image](https://user-images.githubusercontent.com/5090005/219140597-efa79eeb-5606-4228-8267-c7d7a999a54b.png)

# Environment (if applicable)
Version 5.7.1 with OneNote 16.0.16026.20146

# Additional Context

This is my entire XML for the page. I don't know how much it helps. After my trying to understand things, I am instead going to create some separate issues.
```XML
<one:Page xmlns:one="http://schemas.microsoft.com/office/onenote/2013/onenote" ID="{E054B06A-E856-04C3-3A12-DC4DADCDB457}{1}{E181213114545951262291988178892239341527611}" name="Support and Contact info" dateTime="2021-06-07T16:43:18.000Z" lastModifiedTime="2023-02-15T20:07:26.000Z" pageLevel="1" isCurrentlyViewed="true" lang="en-US">
  <one:QuickStyleDef index="0" name="PageTitle" fontColor="automatic" highlightColor="automatic" font="Calibri Light" fontSize="20.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="1" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="2" name="h1" fontColor="#1E4E79" highlightColor="automatic" font="Calibri" fontSize="16.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="3" name="p" fontColor="automatic" highlightColor="automatic" font="Calibri" fontSize="11.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="4" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="5" name="p" fontColor="#000000" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="6" name="p" fontColor="automatic" highlightColor="automatic" font="Calibri" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:PageSettings RTL="false" color="automatic">
    <one:PageSize>
      <one:Automatic />
    </one:PageSize>
    <one:RuleLines visible="false" />
  </one:PageSettings>
  <one:Title lang="en-US">
    <one:OE alignment="left" quickStyleIndex="0">
      <one:T><![CDATA[Support and Contact info]]></one:T>
    </one:OE>
  </one:Title>
  <one:Outline>
    <one:Position x="36.0" y="86.4000015258789" z="0" />
    <one:Size width="789.2814331054687" height="766.2173461914062" isSetByUser="true" />
    <one:OEChildren>
      <one:OE alignment="left" quickStyleIndex="1">
        <one:T><![CDATA[If you have problems with or questions about a product use this list to find the resources you need.]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="2">
        <one:T><![CDATA[Contact Numbers]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1">
        <one:T><![CDATA[I find that calling is my last choice because it often involves a long wait. In addition many companies make it extremely difficult to actually find a phone number. To that end I have started recording any phone numbers that I find/use.]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="3">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left">
        <one:Table bordersVisible="true" hasHeaderRow="true">
          <one:Columns>
            <one:Column index="0" width="49.7955093383789" />
            <one:Column index="1" width="84.84054565429687" />
            <one:Column index="2" width="570.3305053710937" />
          </one:Columns>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[Apple]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[800–275–2273]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[From &lt; <a href="https://www.google.com/search?client=firefox-b-1-d&amp;q=how+to+report+problems+to+apple">https://www.google.com/search?client=firefox-b-1-d&amp;q=how+to+report+problems+to+apple</a>&gt; ]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[Amazon]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[800/201-7575]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[Ebay]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[800/322-9266]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="4">
                  <one:T><![CDATA[]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
        </one:Table>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="3">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="3">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="2">
        <one:T><![CDATA[Submitting Product Bugs and feedback]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left">
        <one:Table bordersVisible="true" hasHeaderRow="true">
          <one:Columns>
            <one:Column index="0" width="139.5414276123047" isLocked="true" />
            <one:Column index="1" width="641.6099853515624" isLocked="true" />
          </one:Columns>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Firefox]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://bugzilla.mozilla.org/show_bug.cgi">https://bugzilla.mozilla.org/show_bug.cgi</a>]]></one:T>
                </one:OE>
                <one:OE alignment="left" quickStyleIndex="5">
                  <one:T><![CDATA[<a href="https://bugzilla.mozilla.org">https://bugzilla.mozilla.org</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Nirvana]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="5">
                  <one:T><![CDATA[support@nirvanahq.com]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[IPhone]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="6">
                  <one:T><![CDATA[<a href="https://www.apple.com/feedback/iphone/">https://www.apple.com/feedback/iphone/</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Microsoft products]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://feedbackportal.microsoft.com/feedback/">https://feedbackportal.microsoft.com/feedback/</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Youtube]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://support.google.com/youtube/answer/4347644">https://support.google.com/youtube/answer/4347644</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Microsoft Security Related]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://msrc.microsoft.com/report/vulnerability/new">https://msrc.microsoft.com/report/vulnerability/new</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Windows 10]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[&lt;win&gt;F  opens the feedback app]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Microsoft 365 apps]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Select Feedback under the File menu]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Apple Feedback]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://www.apple.com/feedback/">https://www.apple.com/feedback/</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Apple Developer Feedback]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://developer.apple.com/bug-reporting/">https://developer.apple.com/bug-reporting/</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[Adobe]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[<a href="https://www.adobe.com/products/wishform.html">https://www.adobe.com/products/wishform.html</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="3">
                  <one:T><![CDATA[]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
        </one:Table>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="3">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="2">
        <one:T><![CDATA[System Status]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left">
        <one:Table bordersVisible="true" hasHeaderRow="true">
          <one:Columns>
            <one:Column index="0" width="37.11000061035156" />
            <one:Column index="1" width="217.7316284179687" />
          </one:Columns>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
                  <one:T><![CDATA[iCloud]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
                  <one:T><![CDATA[<a href="https://www.apple.com/support/systemstatus/">https://www.apple.com/support/systemstatus/</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
        </one:Table>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="6">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="6" style="font-family:Calibri;font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="2">
        <one:T><![CDATA[Questions and Answers]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:Calibri;font-size:11.0pt">
        <one:T><![CDATA[You can always just google for answers, but sometimes you might need to ask a question that you can't find an answer to and these are the places to ask.]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:'Open Sans';font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left">
        <one:Table bordersVisible="true" hasHeaderRow="true">
          <one:Columns>
            <one:Column index="0" width="37.11000061035156" />
            <one:Column index="1" width="164.8970031738281" />
          </one:Columns>
          <one:Row>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="1">
                  <one:T><![CDATA[Apple]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
            <one:Cell>
              <one:OEChildren>
                <one:OE alignment="left" quickStyleIndex="1">
                  <one:T><![CDATA[<a href="http://discussions.apple.com">http://discussions.apple.com</a>]]></one:T>
                </one:OE>
              </one:OEChildren>
            </one:Cell>
          </one:Row>
        </one:Table>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:'Open Sans';font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1" style="font-family:'Open Sans';font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="2" style="font-family:Calibri;font-size:18.0pt;color:#14324D">
        <one:T><![CDATA[References]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="2" style="font-family:'Open Sans';font-size:11.0pt">
        <one:T><![CDATA[]]></one:T>
      </one:OE>
      <one:OE alignment="left" quickStyleIndex="1">
        <one:T><![CDATA[<a href="https://lifehacker.com/how-to-submit-a-bug-report-to-apple-google-facebook-1832277266">https://lifehacker.com/how-to-submit-a-bug-report-to-apple-google-facebook-1832277266</a>]]></one:T>
      </one:OE>
    </one:OEChildren>
  </one:Outline>
</one:Page>
```

**Comment by stevencohn** [comment]
I created a new page to get a page ID, opened the XML dialog, copied your page XML, changing the page ID to my existing page ID and click Update. It saved it and looks OK. The first heading is a standard Heading 1.

I modified my first style to be a (unique font, because I don't have Open Sans installed) and size 18. Pressed Ctrl+Alt+Shift+1 and it applied the style just fine.

Remember that Ctrl+Alt+Shift+1 refers to the _first style in the gallery_, unless you've customized the shortcut in the OneMore Settings. It is not bound to any specifically named style. (I did this on purpose to keep it language agnostic.) Make sure Ctrl+Alt+Shift+1 matches the style you expect it to apply

**Comment by stevencohn** [comment]
@julie777  let me know if I missed a step above. Otherwise, I may need to close this as no-repro

**Comment by julie777** [comment]
This is still a problem. I just had it happen to me again.

However, I think I have more information to add to the issue.

I just applied custom heading one to a paragraph and the font size changed correctly, but when I look at the ribbon it showed the font as Calibri even though my H1 definition is Open Sans.

So I used undo to put everything back to where it was before I started.
The paragraph in question was quickstyle=2 which is normal.
The page existed before I started using OneMore and it already had some items formatted as quickstyle=1, but that wasn't a heading.

In the XML
```
  <one:QuickStyleDef index="0" name="PageTitle" fontColor="automatic" highlightColor="automatic" font="Calibri Light" fontSize="20.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="1" name="p" fontColor="automatic" highlightColor="automatic" font="Calibri" fontSize="11.0" spaceBefore="0.0" spaceAfter="0.0" />
  <one:QuickStyleDef index="2" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
```

Then I format using Custom heading 1 style which is defined as Open sans 18 and the text gets formatted as calibri 18

Paragraph
```
<one:OE alignment="left" quickStyleIndex="3" selected="partial" style="font-family:Calibri;font-size:18.0pt;color:#1E4E79">
```
Definitions
```
<one:QuickStyleDef index="0" name="PageTitle" fontColor="automatic" highlightColor="automatic" font="Calibri Light" fontSize="20.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="1" name="p" fontColor="automatic" highlightColor="automatic" font="Calibri" fontSize="11.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="2" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="3" name="h1" fontColor="#1E4E79" highlightColor="automatic" font="Calibri" fontSize="16.0" spaceBefore="0.0" spaceAfter="0.0" />
```

In order to compare with how OneNote built in styles work I undid the custom style and applied OneNote heading 1

paragraph
```
<one:OE alignment="left" quickStyleIndex="3" selected="partial">
```
definitions
```
<one:QuickStyleDef index="0" name="PageTitle" fontColor="automatic" highlightColor="automatic" font="Calibri Light" fontSize="20.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="1" name="p" fontColor="automatic" highlightColor="automatic" font="Calibri" fontSize="11.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="2" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="3" name="h1" fontColor="#1E4E79" highlightColor="automatic" font="Calibri" fontSize="16.0" spaceBefore="0.0" spaceAfter="0.0" />
```

You can see that OneMore did the same thing as ON, but for some reason got the font wrong. I have restarted OneNote, reloaded my custom style set and tried it multiple times.

This is my custom heading 1 definition.
![image](https://user-images.githubusercontent.com/5090005/226133506-6915ca9d-8e78-4956-9581-8b1d6d9a335a.png)

So the question is, how is OneMore getting the wrong font name?

(I am sorry that I am not set up to debug this right now. I am still recovering from having to install Windows from scratch.)

**Comment by julie777** [comment]
I decided to apply customs styles to the page and see if that fixed things.
I am not sure whether what it did is correct or not. I think it probably isn't.

Given the page had two different paragraph styles defined already I was surprised that it collapsed then into one style.
```
<one:QuickStyleDef index="0" name="PageTitle" fontColor="#000000" highlightColor="automatic" font="Open Sans" fontSize="20.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="1" name="p" fontColor="#000000" highlightColor="automatic" font="Microsoft Sans Serif" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
```

It make sense that it looked for the name="PageTitle" and changed that. I would have expected it to change all of the other specific names (h1, h2, code, etc.) also.

So I did another test because "normal" is a style in OneNote and onenote also allows setting the default font and size. I have the default font and size at Open Sans 12.
I typed a couple lines and then set the second one to ON normal.
```
<one:OE alignment="left" quickStyleIndex="2">
  <one:T><![CDATA[This should be in a normal font]]></one:T>
</one:OE>
<one:OE selected="all" alignment="left" quickStyleIndex="3">
  <one:T selected="all"><![CDATA[This is actually set to normal]]></one:T>
</one:OE>
```
and the definitions are 
```
<one:QuickStyleDef index="0" name="PageTitle" fontColor="automatic" highlightColor="automatic" font="Calibri Light" fontSize="20.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="1" name="p" fontColor="automatic" highlightColor="automatic" font="Calibri" fontSize="11.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="2" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
<one:QuickStyleDef index="3" name="p" fontColor="automatic" highlightColor="automatic" font="Open Sans" fontSize="12.0" spaceBefore="0.0" spaceAfter="0.0" />
```
So it appears that ON doesn't differential ON default font from the ON normal font, unless it is using fixed quickstyle indices.

I am inclined to think that applying custom styles to a page should only change one of the name="p" styles to normal. What are your thoughts?

**Comment by stevencohn** [comment]
So first of all, yes, this is still a bug in 5.8.2. The fix that I _think_ will address this won't be released until 5.8.3, pending.

OK, so quick style indexes are entirely arbitrary and not guaranteed to be the same as changes are made to the page. Generally, OneNote increments the index for each new style applied to the page, so index 2 may be p (paragraph) in one instance but p (cite) in another instance. (Remember that there are only two names, "p" and "h1..h6". So the "p" name covers all other styles.

I've also seen OneNote both keep duplicates instances of exactly the same quick style and collapse duplicate instances down into one quick style. I can only guess why this happens but the point is that the index and whether there are duplicate of a quick style, while that seems inefficient is entirely benign in my experience.
